b2/sync.py

```diff
@@ -545,9 +545,10 @@ def _walk_relative_paths(self, prefix_len, dir_path, reporter):
                 )
             full_path = os.path.join(dir_path, name)
             relative_path = full_path[prefix_len:]
+            # Skip broken symlinks or other inaccessible files
             if not os.path.exists(full_path):
-                # Skip broken symlinks or other inaccessible files
-                reporter.local_access_error(full_path)
+                if reporter:
```

Comment:
I prefer `is not None`, but was thinking you would find the shorter version more Pythonic.  :-)